### PR TITLE
AudioOnlyButton in the overflow menu should be configurable to get a minimalistic ui

### PR DESCRIPTION
**Pull request by trippledave**
I want to make a very simple UI and the AudioOnlyButton in the overflow menu seems redundant since there is also the VideoMuteButton and i am not allowing screensharing.

So i just made it configurable using the feature flags, implementing it like in the ChatButton

btw. just signed the agreement, do you need ID of the document or what? I´m the guy called David, probably should have written down my github profile name?

### CODE REVIEW

**Comment by trippledave** [comment]
requesting status

File: react/features/base/flags/constants.js [hunk 1]
```diff
@@ -124,6 +124,12 @@ export const TOOLBOX_ALWAYS_VISIBLE = 'toolbox.alwaysVisible';
  */
 export const VIDEO_SHARE_BUTTON_ENABLED = 'video-share.enabled';
 
+/**
+ * Flag indicating that the Audio only button in the overflow menu is enabled.
+ * Default: enabled (true).
+ */
+export const AUDIO_ONLY_BUTTON_ENABLED = 'audio-only.enabled';
```
Comment (saghul) [inline react/features/base/flags/constants.js:131]: Please sort it alphabetically.

**Review by saghul — CHANGES_REQUESTED** [review]
Sorry this fell through the cracks. One minor nit, LGTM otherwise.

**Review by trippledave — COMMENTED** [review]
sorted and then fixed merge isssues in
https://github.com/jitsi/jitsi-meet/pull/7944/commits/cbae174bcb6f0101750a4c4ed87cd2c3061b10dc

not sure why it display the lines 9-14 as new, since they where there before but what do i know

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Cheers!

**Pull request merged**